pharmacy_retail/data/pos_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record model="pos.config" id="point_of_sale.pos_config_main" forcecreate="0">
+        <field name="iface_available_categ_ids" eval="[Command.link(ref('pos_category_1')), Command.link(ref('pos_category_2')), Command.link(ref('pos_category_3')), Command.link(ref('pos_category_4')), Command.link(ref('pos_category_5'))]"/>
```

Comment:
From what I understand, this is only to restrict categories. Here, all are allowed, so this is useless.
```suggestion
```